order causing CI failure

It looks like the CI is failing due to order is not set in the db, causing the following:

```
 1) basic-querying find supports non-empty inq:

      AssertionError: expected Array [ 1, 0 ] to equal Array [ 0, 1 ] (at '0', A has 1 and B has 0)
      + expected - actual

       [
      +  0
         1
      -  0
       ]
      
      at Assertion.fail (node_modules/should/lib/assertion.js:92:17)
      at Assertion.Object.defineProperty.value (node_modules/should/lib/assertion.js:164:19)
      at node_modules/loopback-datasource-juggler/test/basic-querying.test.js:488:29
      at tryCatcher (node_modules/bluebird/js/release/util.js:16:23)
      at Promise._settlePromiseFromHandler (node_modules/bluebird/js/release/promise.js:512:31)
      at Promise._settlePromise (node_modules/bluebird/js/release/promise.js:569:18)
      at Promise._settlePromise0 (node_modules/bluebird/js/release/promise.js:614:10)
      at Promise._settlePromises (node_modules/bluebird/js/release/promise.js:693:18)
      at Async._drainQueue (node_modules/bluebird/js/release/async.js:133:16)
      at Async._drainQueues (node_modules/bluebird/js/release/async.js:143:10)
      at Immediate.Async.drainQueues [as _onImmediate] (node_modules/bluebird/js/release/async.js:17:14)
```

We can maybe use a similar solution like:

```
var idName = this.idName(model);
If (!order) {
order = idName;
}
```

cc @jannyHou  @dhmlau

--- a/CODEOWNERS
+++ b/CODEOWNERS
@@ -1,9 +1,9 @@
 # Lines starting with '#' are comments.
 # Each line is a file pattern followed by one or more owners,
-# the last matching pattern has the most precendence.
+# the last matching pattern has the most precedence.
 
 # Alumni maintainers
-# @kjdelisle @loay @ssh24 @virkt25
+# @kjdelisle @loay @ssh24 @virkt25 @b-admike
 
 # Core team members from IBM
-* @jannyHou @b-admike @dhmlau
+* @jannyHou @dhmlau @raymondfeng @hacksparrow @emonddr

